benefits/eligibility/forms.py

```diff
@@ -22,10 +22,11 @@ class EligibilityVerifierSelectionForm(forms.Form):
     # sets label to empty string so the radio_select template can override the label style
     submit_value = _("eligibility.buttons.choose")
 
-    def __init__(self, agency: models.TransitAgency, *args, **kwargs):
+    def __init__(self, agency: models.TransitAgency, classes, *args, **kwargs):
         super().__init__(*args, **kwargs)
         verifiers = agency.eligibility_verifiers.all()
 
+        self.classes = classes
```

Comment:
Is there a more optimal way to declare this optional attribute? @thekaveman

Reply:
I kind of snuck this comment at the bottom of [my other thought](https://github.com/cal-itp/benefits/pull/1023#discussion_r989572808): it might be easier to just hardcode the classes directly in the form, no need to pass any in -- since they are always the same for each form right?

So instead of:

```python
class EligibilityVerifierSelectionForm(forms.Form):
   def __init__(self, agency: models.TransitAgency, classes, *args, **kwargs):
        self.classes = classes

# and later, each time the form is created

form = EligibilityVerifierSelectionForm(classes="offset-lg-1 col-lg-9")
```

We could have:

```python
class EligibilityVerifierSelectionForm(forms.Form):
   def __init__(self, agency: models.TransitAgency, *args, **kwargs):
        self.classes = "offset-lg-1 col-lg-9"
```

It might simplify some of the usage (in the rest of the python code) of the forms?

Reply:
done